docs/vo_service_discovery/vo_service_discovery.rst

```diff
@@ -0,0 +1,202 @@
+.. doctest-skip-all
+
+.. _astroquery.vo_service_discovery:
+
+*************************************************************************
+Virtual Observatory Service Discovery (`astroquery.vo_service_discovery`)
+*************************************************************************
+
+Getting Started
+===============
+
+Archives from around the world have registered their services with the NASA
+Astronomy Virtual Observatory (NAVO) directory. A web interface is available
+at http://vao.stsci.edu/keyword-search/. This module provides methods
+to query this directory by service type, keywords, wavelength band or coverage,
+and source or publisher.
+
+Here is an example showing how to find information about conesearch services
+provided by the NASA Extragalactic Database (NED).
+
+.. code-block:: python
+
+    >>> from astroquery.vo_service_discovery import Registry
+    >>> results = Registry.query(source='ned', service_type='cone')
+    >>> print(results)
+
+                                waveband                         ... service_type
+        -------------------------------------------------------- ... ------------
+                                                        infrared ...   conesearch
+        radio#millimeter#infrared#optical#uv#euv#x-ray#gamma-ray ...   conesearch
+
+The result of the query is an `astropy.table.Table` object. The methods of
+`~astropy.table.Table`
+can be used to inspect the search result.
+
+.. code-block:: python
+
+    >>> print(results.colnames)
+
+        ['waveband', 'short_name', 'ivoid', 'res_description', 'access_url',
+         'reference_url', 'publisher', 'service_type']
+
+The value of ``'access_url'`` specifies the endpoint for the service.
+
+Documentation of the service may be found in the ``'reference_url'`` values.
+You can use a function from the Python standard library to open the second url
+in a web browser.
+
+.. code-block:: python
+
+    >>> import webbrowser
+    >>> webbrowser.open(results['reference_url'][1].decode())
+
+
+Discovering Services
+====================
+
+The `~astroquery.vo_service_discovery.RegistryClass.query` method provides several
+ways to search for specific services of interest. Each of these options
+can be combined into a single call to the method.
+
+The ``service_type`` parameter can be set to the VO service types ``'conesearch'``,
+``'simpleimageaccess'``, ``'simplespectralaccess'``, or ``'tableaccess'``. These
+values may be shortened to ``'cone'``, ``'image'``, ``'spectra'`` or
+``'spectrum'``, or ``'tap'`` or ``'table'``.
+
+The value of the ``keyword`` parameter will return results containg that string
+in the ``ivoid``, ``short_name``, or ``res_description`` fields.
+
+Wavelength range can be specified by a string containing one or more band names.
```

Comment:
this paragraph doesn't define the keyword needed here; presumably it's `wavelength`?

Reply:
no, it's `waveband`.

Reply:
also, maybe combine the last sentence with the first?

Reply:
I reworded the waveband section to try to address the comments.

Regarding the question, VO registries do not currently have a working ability to query for services that have overlap with some part of the sky.  

However, the IVOA is working on enhancements to allow services to describe their sky coverage and to support queries based on that.  Those features would likely be based on Multi-Order Coverage maps, much like the CDS MOC Server does today (https://astroquery.readthedocs.io/en/latest/api/astroquery.cds.CdsClass.html#astroquery.cds.CdsClass.query_region)